### PR TITLE
Updated error-message, mega-modal and tabs with component tests. #trivial

### PR DESCRIPTION
**Pull request by JoshuaNg2332**
f-error-message - v0.4.0
------------------------------
*March 3, 2021*

### Added
- Added component test file and tests

f-mega-modal - v0.5.0
------------------------------
*March 3, 2021*

### Added
- Added component test file, component object file and tests
- Added `Data-test-id` to elements within the `MegaModal.stories.js`

f-tabs - v0.3.0
------------------------------
*March 3, 2021*

### Added
- Added component tests
- Added supporting code to component object file

### Changed
- `Data-test-id` naming for `tab-` to `tab-button-` in `Tabs.vue`
- `Data-test-id` naming for `transition-tab-` to `tab-content-` in `Tab.vue`

### CODE REVIEW

**Review by kevinrodrigues — DISMISSED** [review]

File: packages/components/atoms/f-error-message/CHANGELOG.md [hunk 1]
```diff
@@ -3,8 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-Latest (add to next release)
+v0.3.2
```
Comment (siggerzz) [inline packages/components/atoms/f-error-message/CHANGELOG.md]: Should this be 0.4.0 - We only increment the 3rd digit if it's a bug fix 👍🏼

File: packages/components/atoms/f-error-message/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-error-message",
   "description": "Fozzie Error Message – Generic inline error message",
-  "version": "0.3.1",
+  "version": "0.3.2",
```
Comment (siggerzz) [inline packages/components/atoms/f-error-message/package.json]: Change to 0.4.0

File: packages/components/molecules/f-mega-modal/CHANGELOG.md [hunk 3]
```diff
@@ -3,9 +3,14 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-
-Latest (add to next release)
+v0.4.1
```
Comment (siggerzz) [inline packages/components/molecules/f-mega-modal/CHANGELOG.md]: Change to 0.5.0

File: packages/components/molecules/f-mega-modal/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-mega-modal",
   "description": "Fozzie Mega Modal – A Vue.js modal component",
-  "version": "0.4.0",
+  "version": "0.4.1",
```
Comment (siggerzz) [inline packages/components/molecules/f-mega-modal/package.json]: Change to 0.5.0

File: packages/components/molecules/f-tabs/CHANGELOG.md [hunk 5]
```diff
@@ -3,9 +3,18 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-
-Latest (add to next release)
+v0.2.2
```
Comment (siggerzz) [inline packages/components/molecules/f-tabs/CHANGELOG.md]: Change to 0.3.0

File: packages/components/molecules/f-tabs/package.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@justeat/f-tabs",
   "description": "Fozzie Tabs – Switchable slots for content&#34;",
-  "version": "0.2.1",
+  "version": "0.2.2",
```
Comment (siggerzz) [inline packages/components/molecules/f-tabs/package.json]: Change to 0.3.0

**Review by siggerzz — CHANGES_REQUESTED** [review]
Looking good! Just a few minor comments around the version numbers 🎉

**Review by DamianMullins — DISMISSED** [review]

**Review by siggerzz — APPROVED** [review]

**Review by FayeCarter — APPROVED** [review]

**Pull request merged**